draft/overlay/mbd-pi-hat.md

```diff
@@ -0,0 +1,84 @@
+<!--
+---
+name: MDB HAT
+class: board
+type: io
+formfactor: HAT
+manufacturer: Qibixx
+description: MDB (Machine Data Bus) Bus Interface and Sniffer for the Raspberry Pi
+url: https://qibixx.com
+buy: https://qiba.pt/
+image: 'mbd-pi-hat.png'
```

Comment:
Would it make more sense if the image was called `mdb-pi-hat.png` ? :wink:

Reply:
I had to look at this three times across two separate days before I realised what you meant :D I'll correct it when I merge.